core/conf/server_config.template

```diff
@@ -32,9 +32,9 @@ cache_config:
   cpu_cache_threshold: 0.85         # percentage of data that will be kept when cache cleanup is triggered, must be in range (0.0, 1.0]
   cache_insert_data: false          # whether to load inserted data into cache, must be a boolean
 
-# Skip the following config if you are using GPU version
-  gpu_cache_capacity: 4             # GB, GPU memory used for cache, must be a positive integer
-  gpu_cache_threshold: 0.85         # percentage of data that will be kept when cache cleanup is triggered, must be in range (0.0, 1.0]
+#Uncomment the following config if you are using GPU version
```

Comment:
Uncomment the following config if you are using **CPU** version

Reply:
Are you sure???